ftw/upgrade/testing.py

```diff
@@ -49,6 +52,30 @@ def setUpPloneSite(self, portal):
         applyProfile(portal, 'ftw.upgrade:default')
 
         self.fix_plone_app_jquery_version(portal)
+        self.prevent_csrf_by_initializing_site_storages(portal)
+
+    def prevent_csrf_by_initializing_site_storages(self, portal):
+        """Plone auto-protection results in confirmation pages
+        when first hitting Plone with the browser.
+
+        By hitting the site once we can initialize all standard Plone containers
+        which cause writes on first hit.
+        We do this with disabled protection.
+        """
+        try:
```

Comment:
I don't think so: this is not a test and `unittest.skipIf` just flags the function as far as I understand.
It is also not a problem when the method is executed even when automatic CSRF protection is not active..